### PR TITLE
Add support for Ruby w/o Rails

### PR DESCRIPTION
**Pull request by dblock**
Use ActiveSupport, but no longer require ActionView.

Drop support for Ruby <= 2.2.

Includes #100, #101, #102.

[Build Status](https://travis-ci.org/dblock/distance_of_time_in_words)

### CODE REVIEW

**Comment by dblock** [comment]
Could I maybe have some attention from @radar or @lauranjansen who are the rubygems owners? I am happy to take over and continue maintaining this gem (I'm dblock on rubygems) if you don't want to do it. I'd rather not fork it, but LMK if you'd prefer.

**Comment by radar** [comment]
You opened this PR 7 hours ago and asked for maintainer rights an hour later. It’s Sunday here. I don’t know why you’re doing what you’re doing.

**Comment by dblock** [comment]
Hey sorry if that came the wrong way. I am not asking for anything and definitely not for a faster response. Please obviously prioritize your life and feel free to answer whenever you have time or never answer at all.

The reason I offered to help with maintaining the gem was only that there have not been any commits for a long time and not because I was not getting an answer.

Thanks for your hard work and enjoy your weekend.

**Comment by dblock** [comment]
Also I hope your daughter is feeling better, forgive me for not starting with that.

File: Appraisals [hunk 1]
```diff
@@ -0,0 +1,11 @@
+appraise 'rails_4' do
+  gem 'rails', '~> 4.0'
+end
+
+appraise 'rails_5.0' do
+  gem 'rails', '~> 5.0.0'
+end
+
+appraise 'rails_5.1' do
```
Comment (radar) [inline Appraisals:9]: What about Rails 5.2 + Rails 6.0?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,10 +1,23 @@
...
-* Add CHANGELOG.md
 
 ## 3.1.0 (2016/03/07)
 
-* [#68](https://github.com/radar/dotiw/pull/68): Add support for weeks - [@lauranjansen](https://github.com/lauranjansen)
-* [#69](https://github.com/radar/dotiw/pull/69): Add Indonesian language support - [@avidmaulanas](https://github.com/avidmaulanas)
-* [#70](https://github.com/radar/dotiw/pull/70): Add French language support - [@geo1004](https://github.com/geo1004)
-* [#72](https://github.com/radar/dotiw/pull/72): Add Danish language support - [@kaspernj](https://github.com/kaspernj)
-* [#71](https://github.com/radar/dotiw/pull/71): Update bundler and ruby versions for Travis CI - [@lauranjansen](https://github.com/lauranjansen)
+* [#68](https://github.com/radar/dotiw/pull/68): Add support for weeks - [@lauranjansen](https://github.com/lauranjansen).
```
Comment (radar) [inline CHANGELOG.md:11]: Thank you for adding these.

File: README.markdown [hunk 3]
```diff
@@ -1,175 +1,246 @@
-# dotiw [![](https://travis-ci.org/radar/dotiw.svg)](https://travis-ci.org/radar/dotiw)
+# dotiw
```
Comment (radar) [inline README.markdown:2]: Please remove these README changes from this PR.

**Comment by radar** [comment]
Note that the Ruby 2.4.x build is currently failing here too.

**Comment by radar** [comment]
I've also just added you as an owner to this gem on RubyGems and invited you as a collaborator here. Thank you for volunteering!

File: Appraisals (hunk 1 above)
Comment (dblock) [reply Appraisals:9]: They weren't here before, added in #104.

**Comment by dblock** [comment]
> Note that the Ruby 2.4.x build is currently failing here too.

This is https://stackoverflow.com/questions/41703128/error-while-trying-to-load-the-gem-devise-activesupport-duration-cant-be-coe, which I couldn't fix easily. I propose to drop 2.4.

**Comment by dblock** [comment]
> Please add support for the additional Rails versions (5.2 + 6.0). Along with this, please configure the Github actions to use the appraisal files. Here's how I've done it on the i18n project: https://github.com/ruby-i18n/i18n/blob/master/.github/workflows/ruby.yml.

Done in #104

**Comment by dblock** [comment]
Superseded by https://github.com/radar/distance_of_time_in_words/pull/105

**Pull request closed without merging**